src/App.jsx

```diff
@@ -12,7 +13,7 @@ export default function App() {
   const [taskTitle, setTaskTitle] = useState('');
 
   const taskCount = list.length;
-  const completedCount = list.reduce((count, task) => task.completed ? count + 1 : count, 0);
+  const completedCount = list.filter(task => task.completed).length;
```

Comment:
simplify this, usually `.reduce()` is used for transforming an array of objects to an object , counting the completed tasks may be achieved filtering the completed and count those